india_compliance/gst_india/api_classes/nic/e_waybill.py

```
@@ -0,0 +1,195 @@
+        if self.settings.enable_e_waybill:
+            return
+
+        frappe.throw(_("Please enable e-Waybill features in GST Settings first"))
+
+    def post(self, action, json):
+        return super().post(params={"action": action}, json=json)
+
+    def get_transporter_details(self, transporter_id):
+        return self.get("GetTransporterDetails", params={"trn_no": transporter_id})
+
+    def is_ignored_error(self, response_json):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)